scripts/make_db.py

```diff
@@ -1,79 +0,0 @@
-from pokemon.convert import handle_image_conversion
-from BeautifulSoup import BeautifulSoup
```

Comment:
Ah I see - I think it would be best if the make_db is still treated as a script (in this location) and not as a part of the module.

Reply:
Absolutely; I'll change back.